code/controllers/subsystem/tcgsetup.dm

```diff
@@ -0,0 +1,10 @@
+#define CARD_FILES list("set_one.json","set_two.json")
+#define CARD_DIRECTORY "strings/tcg"
+#define CARD_PACKS list(/obj/item/cardpack/series_one, /obj/item/cardpack/resin)
+SUBSYSTEM_DEF(trading_card_game)
+	name = "Trading Card Game"
+	flags = SS_NO_FIRE
+
+/datum/controller/subsystem/trading_card_game/Initialize()
+	reloadAllCardFiles(CARD_FILES, CARD_DIRECTORY)
+	. = ..()
```

Comment:
Does `reloadAllCardFiles(CARD_FILES, CARD_DIRECTORY)` need to happen before calling the parent?

Reply:
Calling the parent where the "we completed loading X in X seconds" print happens.

I'm actually a little worried about using Initialize(), because it means we can't load cards that have been placed on a map at roundstart

Reply:
Add a proc then, called on the parent and redefined on the child.